package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -394,6 +394,7 @@ public string usdFullPath
           stage = pxr.UsdStage.Open(usdFullPath, pxr.UsdStage.InitialLoadSet.LoadAll);
         }
 
+        stage.Reload();  // Ensure the stage is reloaded it case it was already opened and cached.
```

Comment:
small typo: `reloaded it` should be `reloaded in`

Reply:
Fixed in https://github.com/Unity-Technologies/usd-unity-sdk/pull/216/commits/6aa01e5a0a82d457c391f5056b885b1fbf7f833a